### PR TITLE
[CORL-435] Stable CSS Classes

### LINKED ISSUE #2377: [Next] Custom CSS
Hello, we are working with Coral Talk (next branch) and we feel it’s hard to customize the default embed stream using the **Custom CSS** feature. It would be nice if the components had more friendly selectors. What do you think? I can help with a PR if necessary.

Thank you

### PR DESCRIPTION
**Pull request by wyattjoh**
## What does this PR do?

- Fixes import issues within the client codebase
- Introduces stable CSS classes in [classes.ts](https://github.com/coralproject/talk/blob/feature/CORL-435/src/core/client/stream/classes.ts) where the class names are assigned there

To show off where the customizable zones are, you can set the following CSS rule:

```css
.coral {
  border: 1px solid coral;
}
```

- Resolves #2377 

Which will outline all the customization zones with borders.

## How do I test this PR?

1. `npm run build`
2. `npm run start`
3. Create a custom CSS file, host it somewhere (locally using [serve](https://www.npmjs.com/package/serve) or similar), containing the above mentioned CSS to highlight customization zones

### CODE REVIEW

**Comment by wyattjoh** [comment]
This should resolve #2377.

File: src/core/client/stream/test/comments/stream/__snapshots__/loadMore.spec.tsx.snap [hunk 1]
```diff
@@ -75,7 +76,7 @@ exports[`renders comment stream with load more button 1`] = `
                     className="Box-root Flex-root Flex-flex Flex-itemGutter Flex-alignBaseline Flex-directionRow gutter"
                   >
                     <time
-                      className="Timestamp-root RelativeTime-root"
+                      className="Timestamp-root coral coral-comment-timestamp RelativeTime-root"
```
Comment (nick-funk) [inline src/core/client/stream/test/comments/stream/__snapshots__/loadMore.spec.tsx.snap]: May want to move the `coral coral-comment-timestamp` to the end of this so they can override `RelativeTime-root` if they wanted to.

File: src/core/client/stream/test/comments/stream/__snapshots__/renderMessageBox.spec.tsx.snap [hunk 2]
```diff
@@ -366,7 +367,7 @@ exports[`renders message box when logged in 1`] = `
             className="Box-root Flex-root Flex-flex Flex-alignFlexEnd Flex-directionColumn"
           >
             <button
-              className="BaseButton-root Button-root Button-sizeRegular Button-colorPrimary Button-variantFilled Button-disabled"
+              className="BaseButton-root Button-root coral coral-createComment-submit Button-sizeRegular Button-colorPrimary Button-variantFilled Button-disabled"
```
Comment (nick-funk) [inline src/core/client/stream/test/comments/stream/__snapshots__/renderMessageBox.spec.tsx.snap]: Order of classnames might matter here.

File: src/core/client/stream/test/profile/__snapshots__/myComments.spec.tsx.snap [hunk 3]
```diff
@@ -20,7 +20,7 @@ exports[`renders profile 1`] = `
       Story: title
     </h1>
     <time
-      className="Timestamp-root RelativeTime-root"
+      className="Timestamp-root coral coral-comment-timestamp RelativeTime-root"
```
Comment (nick-funk) [inline src/core/client/stream/test/profile/__snapshots__/myComments.spec.tsx.snap]: Order of classnames may matter here.

**Review by kgardnr — APPROVED** [review]

**Pull request merged**